packages/steps/src/__tests__/BuildConfig-test.ts

```diff
@@ -458,6 +465,42 @@ describe(validateConfig, () => {
           validateConfig(BuildConfigSchema, buildConfig);
         }).not.toThrow();
       });
+
+      test('valid allowed platforms for function', () => {
+        const buildConfig = {
+          build: {
+            steps: ['abc'],
+          },
+          functions: {
+            abc: {
+              platforms: ['linux', 'darwin'],
+              command: 'echo "${ inputs.i1 } ${ inputs.i2 }"',
```

Comment:
It doesn't matter because these are tests but let's change it a bit so the test example makes more sense (the function doesn't actually have inputs).
```suggestion
              command: 'echo "abc"',
```